elastic/synthetics-recorder
# [Upgrade] Upgrade Synthetics to v1.0.0 and migrate code gen from Recorder to Synthetics

## Summary

Resolves https://github.com/elastic/synthetics-recorder/issues/295.

Resolves https://github.com/elastic/synthetics-recorder/issues/393.

Upgrades the recorder to use v1.0.0 of Synthetics 🎉 

Migrates code generation to the synthetics library.

~In draft state, code not ready for review.~

All checks are passing, this should be ready to review.

## Implementation details

Bumps both Synthetics and Playwright to the version used by Synthetics.

Deletes code gen from the recorder and references updated code in Synthetics lib.

## How to validate this change

Make sure the recorder works as intended, specifically WRT generated code.

- Download and smoke test the recorder
- Record multiple journeys
- Make sure code export works ok
- Ensure the "test" feature of the recorder is working and instrumenting the browser properly
